### PR TITLE
Duplicate files

### PR DESCRIPTION
**Pull request by ulrichmathes**
* remove non helpful header
* only show preview images on images
* open folder with search term
* incease thumnail size

### CODE REVIEW

File: Classes/Widgets/DuplicateFilesWidget.php [hunk 1]
```diff
@@ -14,23 +13,24 @@
...
         $queryBuilder = $this->connectionPool->getConnectionForTable('sys_file')->createQueryBuilder();
         $queryBuilder->getRestrictions()->removeAll();
 
         $duplicates = $queryBuilder
             ->selectLiteral('GROUP_CONCAT(uid) as uids', 'sha1', 'count(*) as counting')
             ->from('sys_file')
-            ->addOrderBy('counting', 'desc')
+            ->orderBy('counting', 'desc')
+            ->addOrderBy('size', 'desc')
             ->where('missing = 0 AND storage > 0 AND name != "index.html" AND identifier NOT LIKE "%_recycler_%"')
```
Comment (Kayu84) [inline Classes/Widgets/DuplicateFilesWidget.php:34]: ->where('missing = 0 AND storage > 0 AND name != "index.html" AND identifier NOT LIKE "%_recycler_%"')

`missing = 0` only works if index scheduler is executed.
I got errors where file is not marked as missing but Folder doesn't exists anymore.

Comment (ulrichmathes) [reply Classes/Widgets/DuplicateFilesWidget.php:34]: then, we have to fix the folder errors as missing = 0 is default for sys_file

File: Classes/Widgets/DuplicateFilesWidget.php [hunk 2]
```diff
@@ -14,23 +13,24 @@
...
             ->groupBy('sha1')
             ->having('counting > 1')
```
[the remark this answers is not in the document]
Comment (ulrichmathes) [reply Classes/Widgets/DuplicateFilesWidget.php:36]: i'll go with 200

**Comment by ulrichmathes** [comment]
> Existing *.webp files doesn't get a preview image.

If webp is handled as an Image and therefore gets an preview image is based on `$GLOBALS['TYPO3_CONF_VARS']['GFX']['imagefile_ext']` and may not be configured that way in your project.

**Pull request merged**